app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -73,6 +84,11 @@ private void initViews() {
         appListRecyclerView.setLayoutManager(layoutManager);
 
         textViewLastSynchronization = findViewById(R.id.textViewLastSynchronization);
+        progressBarLoading = findViewById(R.id.progress_bar_loading);
+
+        //Show loading dialog
+        progressBarLoading.setVisibility(View.VISIBLE);
```

Comment:
Is this line necessary if the XML view is visible by default?

Reply:
No it's not. I just removed it.